app/static/js/_navbar.js

```diff
@@ -10,10 +10,74 @@ const minEditZoom = 13
 const navbar = document.querySelector(".navbar")
 const editGroup = navbar.querySelector(".edit-group")
 const loginLinks = navbar.querySelectorAll("a[href='/login']")
-
-// Configure the remote edit button (JOSM)
 const remoteEditButton = navbar.querySelector(".remote-edit")
-if (remoteEditButton) configureRemoteEditButton(remoteEditButton)
+
+const editButtons = editGroup.querySelectorAll(".dropdown-item.edit-link")
+const rememberChoice = editGroup.querySelector("input[name='remember-choice']")
+
+if (rememberChoice.disabled) {
+    Tooltip.getOrCreateInstance(rememberChoice.parentElement, {
+        title: rememberChoice.parentElement.dataset.bsTitle,
+        placement: "bottom"
+    }).enable();
+}
```

Comment:
Instead of writing JS code, let's put this in the HTML template: https://getbootstrap.com/docs/5.3/components/tooltips/#directions